tests/server-integration/event-client.tap.ts

```diff
@@ -0,0 +1,104 @@
+import test from 'tape'
+
+import { verifyNrIntegrationErrors, NewRelicFeature } from './insights-results'
+
+import {
+  EventClient,
+  EventClientOptions,
+  EventBatch,
+  Event
+} from '../../src/telemetry/events'
+
+const eventConfig: EventClientOptions = {
+  apiKey: process.env.TEST_API_KEY,
+  host: process.env.TEST_EVENT_HOST
+}
+
+test('Event Client - Server Integration Tests', (t): void => {
+  t.ok(eventConfig.apiKey, 'TEST_API_KEY must be configured for tests')
+
+  t.test('Should send batch of individually added events', (t): void => {
+    const attributes = {
+      foo: 'bar'
+    }
+
+    const batch = new EventBatch(attributes)
+
+    const event1: Event = {
+      eventType: 'TestEvent',
+      timestamp: Date.now(),
+      attributes: {
+        foo: 'baz',
+        bar: 'baz',
+      }
+    }
+
+    const event2: Event = {
+      eventType: 'AnotherTestEvent',
+      timestamp: Date.now(),
+      attributes: {
+        foo: 'bar',
+      }
+    }
+
+    batch.addEvent(event1)
+    batch.addEvent(event2)
+
+    const client = new EventClient(eventConfig)
+
+    client.send(batch, (err, res, body): void => {
+      t.error(err)
+      t.ok(res)
+      t.ok(body)
+
+      t.equal(res.statusCode, 202)
```

Comment:
Looks like this endpoint isn't on the same processing as the others and returns 200 instead of 202.

```suggestion
      t.equal(res.statusCode, 200)
```